Sources/OTel/OTel+Bootstrap.swift

```diff
@@ -20,6 +20,94 @@ import Tracing
+    /// - Returns: A service that manages the background work for the bootstrapped observability backends.
+    ///
+    ///   The returned service manages the complete lifecycle of all observability backends. It handles startup,
+    ///   background processing, and graceful shutdown of exporters and processors.  The service performs these
+    ///   operations.
+    ///
+    ///   > Important: You must run this service in a `ServiceGroup` alongside your application services for
+    ///   observability data to be exported.
+    ///
+    /// This is the primary API for setting up observability in your Swift application. Call this function once during
+    /// application startup to configure logging, metrics, and tracing with OTLP exporters.
+    ///
+    /// This function bootstraps the process-global observability subsystems that are enabled in the configuration.
+    /// Attempting to bootstrap these subsystems multiple times will result in a fatal error. If you wish to bootstrap
+    /// only a subset of the observability subsystems, provide a configuration with only those subsystems enabled.
+    ///
+    /// This API supports overriding the configuration using environment variables defined in the OpenTelemetry
+    /// specification. This enables operators to customize the observability of your application during deployment.
+    /// For more details on the configuration options, their defaults, and their associated environment variables, see
+    /// `OTel.Configuration`.
+    ///
+    /// If you need greater control over the bootstrap of the observability subsystems, use the APIs that return the
+    /// backends themselves, for example, `OTel.makeTracingBackend(configuration:)`.
+    ///
+    /// ## Example usage
+    ///
+    /// ### Bootstrap observability backends with minimal ceremony
+    ///
+    /// To bootstrap all the observability backends with default configuration, simply call `OTel.bootstrap` with no
+    /// parameters and run the returned service along with your application service.
+    ///
+    /// ```swift
+    /// // Bootstrap observability backends and get a single, opaque service, to run.
+    /// let observability = try OTel.bootstrap()
+    ///
+    /// // Run observability service(s) in a service group with adopter service(s).
+    /// let server = MockService(name: "AdopterServer")
+    /// let serviceGroup = ServiceGroup(services: [observability, server], logger: .init(label: "ServiceGroup"))
+    /// try await serviceGroup.run()
+    /// ```
+    ///
+    /// ### Configure/disable observability backends
+    ///
+    /// To configure the behavior of the observability backends, first start with the default configuration and override
+    /// the properties as required.
+    ///
+    /// - Note: some of the values in this example are the default, but are explicitly set for illustrative purposes.
+    /// - Note: additional configuration will be applied from the environment variables defined in the OTel spec.
```

Comment:
```suggestion
    /// - Note: Some of the values in this example are the default, but are explicitly set for illustrative purposes.
    /// - Note: Additional configuration will be applied from the environment variables defined in the OTel spec.
```

Nit: I believe this should be sentence cased.